checkout/interfaces/controller/checkoutcontroller.go

```diff
@@ -248,6 +248,9 @@ func (cc *CheckoutController) PlaceOrderAction(ctx context.Context, r *web.Reque
 	if err != nil {
 		cc.logger.WithContext(ctx).WithField("subcategory", "checkoutError").WithField("errorMsg", err.Error()).Error(fmt.Sprintf("place order failed: cart id: %v / total-amount: %v", decoratedCart.Cart.EntityID, decoratedCart.Cart.GrandTotal()))
 		if paymentError, ok := err.(*paymentDomain.Error); ok {
+			if paymentError.ErrorCode == "payment-aborted-by-customer" {
+				return cc.showCheckoutFormAndHandleSubmit(ctx, r, "checkout/checkout")
+			}
```

Comment:
Could you move the "payment-aborted-by-customer" to a constant?

Reply:
Created a constant for this now.

Reply:
Moved constant to payment domain.